### DISCUSSION TITLE
No host selected

### DISCUSSION THREAD

**Discussion by adithyasrinivasan** [deployphp/deployer#2387] [Help needed] [answered]
Hi,

I'm running deployer on Ubuntu-latest for PHP8 on 7.0.13-beta. This is my definition on the file.

```

host('IP_OF_MY_SITE')
    ->setHostname('IP_OF_MY_SITE')
    ->setLabels(['stage' => 'production'])
    ->setRemoteUser('root')
    ->set('deploy_path', '/var/www/src');

```

Unfortunately, I'm getting the following error. 
```

In SelectCommand.php line 72:
                                  
  [Deployer\Exception\Exception]  
  No host selected           
```

I read the upgrade guide but is there anything that I'm missing? Thanks a ton!

**Comment by adithyasrinivasan** [comment, marked as answer]
When I ran the `dep deploy production`  I had the alias wrong. Hope this helps someone!

**Reply by antonmedv** [reply]
You can install Deployer autocomplete. It supports host aliases.
